tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/publicAsset/PublicAssetSetService.scala

```diff
@@ -0,0 +1,40 @@
+package com.linagora.tmail.james.jmap.publicAsset
+
+import jakarta.inject.{Inject, Named}
+import org.apache.commons.configuration2.Configuration
+import org.apache.james.jmap.api.model.{IdentityId, UploadId}
+import org.apache.james.jmap.core.AccountId
+import org.apache.james.jmap.mail.BlobId
+import org.apache.james.jmap.method.IdentityResolver
+import org.apache.james.mailbox.MailboxSession
+import reactor.core.scala.publisher.{SFlux, SMono}
+
+import scala.util.Try
+
+class PublicAssetSetService @Inject()(val identityResolver: IdentityResolver,
+                                      @Named("jmap") jmapConfiguration: Configuration) {
+
+  private val publicURIPrefix: String = jmapConfiguration.getString("publicAsset.publicURIPrefix", "")
```

Comment:
IMO we can reuse `url.prefix` cf https://github.com/apache/james-project/blob/master/src/site/xdoc/server/config-jmap.xml#L61

e.g. on staging env `TMAIL_JMAP_URL_PREFIX : https://apisix.upn.integration-open-paas.org`.

Reply:
nice, I like it
I updated as your suggestion https://github.com/linagora/tmail-backend/pull/1057/commits/933d890c308667940329c122b00a58809a4969a7